front/.eslintrc.js

```diff
@@ -25,14 +18,17 @@
     */
     "unused-imports/no-unused-imports": "error",
     "react/no-unescaped-entities": 0,
+    "@typescript-eslint/no-explicit-any": 0,
```

Comment:
Don't want to address these right now. Some places in the code have types that are not obvious to fix.
Lets take care of that in another PR. Soon or not, we'll see.